api/v1/deployment_types.go

```diff
@@ -29,7 +29,7 @@ type DeploymentStatus struct {
 }
 
 // +kubebuilder:object:root=true
-
+// +kubebuilder:resource:shortName={csi,csis}
```

Comment:
Maybe should shortnames be added into CRD too, if we don't regenerate it?

Reply:
Fixed